.github/workflows/build.yml

```diff
@@ -11,10 +11,16 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Enable Corepack and Install Correct Yarn Version
+        run: |
+          corepack enable
+          yarn set version 4.0.2
```

Comment:
We might convert to bun soon but for now we will leave this since it passes build thanks!